api/src/ballots.py

```diff
@@ -15,7 +15,32 @@
 
 @ballots_bp.route('/', methods=['GET'])
 def ballots():
- 
+  # perform the query based off given parameters
+  cur = perform_query()
```

Comment:
`ballots.py` was updated in a newer commit, so this PR needs to include those changes in here before we can merge it

Reply:
fixed